tests/test_us_verification.py

```diff
@@ -24,7 +24,7 @@ def test_us_verification_override_lob_version(self):
             city='San Francisco',
             state='CA',
             zip_code='94107',
-            lob_version='2018-03-30'
```

Comment:
This was causing a test to fail `InvalidRequestError: Lob-Version 2018-03-30 is no longer supported.`.